test/end-to-end/pages/signout/signout.js

```diff
@@ -1,14 +1,9 @@
 'use strict';
 
 const pageUnderTest = require('app/steps/ui/signout');
-const testConfig = require('config');
 
 module.exports = async function() {
     const I = this;
-
-    await I.seeCurrentUrlEquals(pageUnderTest.getUrl());
-
-    await I.waitForText('sign back in', testConfig.TestWaitForTextToAppear);
-
+    await I.seeInCurrentUrl(pageUnderTest.getUrl());
     await I.navByClick('#main-content > div > div > p:nth-child(3) > a');
```

Comment:
this could leave an issue for tests running in Safari, because it responds with the current URL differently to other browsers. Can you put the `waitForText()` back in here to make it more robust